### DISCUSSION TITLE
Can a computed column appear as non-nullable in the GraphQL schema?

### DISCUSSION THREAD

**Discussion by carlpaten** [hasura/graphql-engine#7869] [Question]
We have Postgres functions that can never possibly return NULL and we're using them to populate computed columns. However in the GraphQL schema the fields appear as nullable.

Is there any way to mark them as non-nullable instead?

**Comment by danielboros1990** [comment]
> We have Postgres functions that can never possibly return NULL and we're using them to populate computed columns. However in the GraphQL schema the fields appear as nullable.
> 
> Is there any way to mark them as non-nullable instead?

@carlpaten did find any solution since then?
